### PR TITLE
Feature/seedpeer engine integration

### PR DESCRIPTION
**Pull request by devonhk**
seedpeer.eu torrent search engine integration. I'm not sure what the policy is on search engines that don't support `https`. If that's an issue let me know.

### CODE REVIEW

File: searx/engines/seedpeer.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        ages = dom.xpath(age_xpath)
+    else:  # under ~5 results uses a different xpath
+        torrent_links = dom.xpath(alternative_torrent_xpath)
+        seeds = dom.xpath(alternative_seeds_xpath)
+        peers = dom.xpath(alternative_peers_xpath)
+        titles = dom.xpath(alternative_title_xpath)
+        sizes = dom.xpath(alternative_size_xpath)
+        ages = dom.xpath(alternative_age_xpath)
+    # return empty array if nothing is found
+    if not torrent_links:
```
Comment (kvch) [inline searx/engines/seedpeer.py]: This should be moved under line 47, to run on fastpath if the `torrent_links` is `None`.

**Comment by asciimoo** [comment]
nice.
Please set it disabled by default in settings.yml if it cannot use https.

**Comment by devonhk** [comment]
@asciimoo seedpeer.eu has been disabled by default. Also sorry about the commit history, I'm not completely conformable with rebasing.

**Review by kvch — APPROVED** [review]

**Comment by asciimoo** [comment]
@pydo, thanks.
Please specify the default categories in the engine (`categories = ['files', 'music', 'videos']`) or in settings.yml (`categories : files.music,videos`)

**Comment by devonhk** [comment]
@asciimoo default categories have been set. Hopefully I haven't forgotten anything else.

**Comment by asciimoo** [comment]
thanks!

**Pull request merged**